src/components/fields/SchemaField.js

```diff
@@ -149,7 +150,8 @@ function SchemaFieldRender(props) {
     return <div />;
   }
 
-  let displayLabel = true;
+  const uiOptions = getUiOptions(uiSchema);
+  let { label: displayLabel } = uiOptions;
```

Comment:
Yeah, it did work because the default value is set in `DefaultTemplate.defaultProps`. 

Anyways, it's clearer for the reader if the default value is also set here. I made the necessary changes to the code.